webapp/src/components/App/Singlestudy/explore/Modelization/Areas/Hydro/utils.ts

```diff
@@ -156,3 +165,31 @@ function generateColumns(columnSuffix = ""): string[] {
   }
   return columns;
 }
+
+/**
+ * A custom hook that returns area options filtered by the given fields.
+ *
+ * @param fields - An array of objects containing areaId and coefficient.
+ * @returns A filtered list of AreaOptions.
```

Comment:
"A filtered list of DynamicList's options."

But we already see the return type in TS...